projects/batfish-client/pom.xml

```diff
@@ -137,6 +138,11 @@
     </dependency>
 
     <dependency>
+      <groupId>org.glassfish.jersey.core</groupId>
+      <artifactId>jersey-server</artifactId>
```

Comment:
Why is this not a test scope dependency? Client ought not to depend on service in compile scope, we hope.